screens/Contacts/ContactListScreen.tsx

```diff
@@ -0,0 +1,124 @@
+  {uid: '2', firstName: 'Stephano', lastName: 'Carasto', description: 'No description', interests: [], qualifications: ['surfing', 'machine learning']},
+  {uid: '3', firstName: 'Henrique', lastName: 'Nique', description: 'Good question', interests: ['surfing'], qualifications: ['machine learning']},
+  {uid: '4', firstName: 'Hervé', lastName: 'Delamontagne', description: 'Description, description, description, loooooooooong descriiiiiiiiiiption, this is a veeeeeeeeeeeeeeeeeeeeeery loooooooooooooooooooooooooooooooooooooooooooong descriiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiption alala', interests: ['running'], qualifications: ['bowling']},
+  {uid: '5', firstName: 'Charles', lastName: 'Dupond', description: 'Aaaaaaaaaaaaaaaaaaaa', interests: ['machine learning'], qualifications: ['running', 'bowling']},
+  {uid: '6', firstName: 'Charlotte', lastName: 'Motte', description: 'Bbbbbbbbbbbbbbbbbb', interests: ['walking'], qualifications: ['running']},
+  {uid: '7', firstName: 'Charlović', lastName: 'Poutine', description: 'Description 7', interests: [], qualifications: ['python', 'java', 'C++']},
+  {uid: '8', firstName: 'Stephanović', lastName: 'Vladimir', description: 'I love fish', interests: ['surfing', 'machine learning'], qualifications: ['biology']},
+  {uid: '9', firstName: 'Lulu', lastName: 'Nom', description: 'I love bamboo', interests: ['surfing', 'machine learning'], qualifications: ['litterature']},
+  {uid: '10', firstName: 'Lili', lastName: 'De', description: '19 + 4 = 22', interests: ['surfing', 'machine learning'], qualifications: ['a', 'b', 'c', 'd', 'e', 'f', 'h']},
+  {uid: '11', firstName: 'Lala', lastName: 'Famille', description: 'I\'m doing university for fun', interests: ['a', 'b', 'c', 'd', 'e', 'f', 'h'], qualifications: ['a', 'b', 'c', 'd', 'e', 'f', 'h']},
+  {uid: '12', firstName: 'Abc', lastName: 'Onu', description: 'Didn\'t want a description', interests: [], qualifications: ['a', 'b', 'c', 'd', 'e', 'f', 'h']},
+  {uid: '13', firstName: 'Def', lastName: 'Steph', description: '-', interests: ['movies'], qualifications: ['history']},
+  {uid: '14', firstName: 'Hij', lastName: 'Non', description: 'Mataphisical question', interests: [], qualifications: []},
+]
+
+export const ContactListScreen = () => {
+  const [filteredContacts, setFilteredContacts] = useState(dummyData);
+  const [searchText, setSearchText] = useState('');
+  const [selectedTab, setSelectedTab] = useState('Plain View');
+
+  const handleSearch = (text: string) => {
+    setSearchText(text);
+    if(text){
+        const filtered = dummyData.filter((contact) =>
+          `${contact.firstName} ${contact.lastName}`.toLowerCase().includes(text.toLowerCase())
+        )
+        setFilteredContacts(filtered);
+    }else{
+        setFilteredContacts(dummyData);
+    }
+  }
+
+  const TopScreen = () => { // will be a module
+    return (
+      <View style={styles.horizontalContainer}>
+        <Image 
```

Comment:
You probably missed this when we talked about this in-person, but importing a `.png` or a `.svg` for an icon is not efficient. You should install and use `@expo/vector-icons` package which provides you with many icons out of the box. Also, all of the icons are stylable. You can find all icons listed [here](https://icons.expo.fyi/Index). E.g. you would probably want to use https://icons.expo.fyi/Index/Ionicons/location-outline from Ionicons here.

1. Import it in your file with `import { Ionicons } from '@expo/vector-icons'`
2. Use it and style the icon as you wish by rendering `<Ionicons name="location-outline" size={24} color="black" />`